lib/screens/authenticate/register.dart

```diff
@@ -0,0 +1,111 @@
+import 'package:brew_crew/shared/loading.dart';
+import 'package:flutter/material.dart';
+import 'package:brew_crew/services/auth.dart';
+import 'package:brew_crew/shared/constans.dart';
+
+class Register extends StatefulWidget {
+  final Function toggleView;
+
```

Comment:
_toggleView

Reply:
tutaj jest to funkcja którą przyjmuje w konstruktorze, więc nie mogę użyć znaku _